Dockerfile.kmm-operator-build

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18
+FROM registry.ci.openshift.org/openshift/release:golang-1.19
```

Comment:
@yevgeny-shnaidman Is this intended? Why are we using `registry.ci.openshift.org/openshift/release:golang-1.18` u/s?

Reply:
yes, you are correct. I am closing this PR and will open a new one with the fix